proto/leader.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package leader;
+
+import "common.proto";
+
+message LeaderRequest {}
+
+message LeaderResponse {
```

Comment:
FYI, in order to implement service discovery in `meta_client`, I think we need more informations rather than just leader address information. For example if we scale-in some new meta nodes and then shutdown all previous ones. We still can achieve service discovery without restarting other components using `meta_client`. For now it's okay to define the API like this and it's quit hard to resolve all candidates in current implementation. And I have discuss it with @shanicky , he will expose that information in his PR.

Reply:
Lets discuss this in slack together with the others. I agree that we need more to handle meta failover successfully.